packages/textlint-formatter/src/textlint-formatter.ts

```diff
@@ -0,0 +1,44 @@
+// LICENSE : MIT
+"use strict";
+import { TextlintResult } from "@textlint/kernel";
+
+const fs = require("fs");
+const path = require("path");
+const tryResolve = require("try-resolve");
+const interopRequire = require("interop-require");
+const isFile = require("is-file");
+const debug = require("debug")("textlint:textfix-formatter");
+
+export type FormatterConfig = { color?: boolean; formatterName: string };
+
+export function createFormatter(formatterConfig: FormatterConfig) {
```

Comment:
Yeah.

📝  It is BREAKING CHANGE.

Tips: Commit message of BREAKING CHANGE.

> A description MUST be provided after the BREAKING CHANGE: , describing what has changed about the API, e.g., BREAKING CHANGE: environment variables now take precedence over config files.  
> https://conventionalcommits.org/

```
chore(textlint): import instead of require

BREAKING CHANGE: export `createFormatter` instead of `default`
```

Reply:
Exactly. This is my first breaking change in any OSS 😄 
Do I need to amend my commit message by rebasing?

Reply:
>  Do I need to amend my commit message by rebasing?

If possible!

Thanks for consideration.

Reply:
@azu, I'm sorry but I found that my rewording to the commit had lost for unknown reason. It might be due to my mistake while tinkering git branches...